### PR TITLE
Buffer exchange

### PR DESCRIPTION
**Pull request by gh-dkale**
Tickets resolved: [EPIR-243](https://guardanthealth.atlassian.net/jira/software/c/projects/EPIR/boards/742?assignee=608779c7ccda29006b5cd59d&selectedIssue=EPIR-243), [EPIR-256](https://guardanthealth.atlassian.net/jira/software/c/projects/EPIR/boards/742?assignee=608779c7ccda29006b5cd59d&selectedIssue=EPIR-256) for buffer exchange workflow with 1 and 2 batches and [EPIR-257](https://guardanthealth.atlassian.net/jira/software/c/projects/EPIR/boards/742?assignee=608779c7ccda29006b5cd59d&selectedIssue=EPIR-257), [EPIR-258](https://guardanthealth.atlassian.net/jira/software/c/projects/EPIR/boards/742?assignee=608779c7ccda29006b5cd59d&selectedIssue=EPIR-258 ) for MBD Setup workflow with 1 and 2 batches with ACS Platform `master-9536-b833d9956` and pipeline `Epi Only (master-654-afde5aef8)` with initial setup of the environment. 

[EPIR-243]: https://guardanthealth.atlassian.net/browse/EPIR-243?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EPIR-256]: https://guardanthealth.atlassian.net/browse/EPIR-256?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[EPIR-257]: https://guardanthealth.atlassian.net/browse/EPIR-257?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EPIR-258]: https://guardanthealth.atlassian.net/browse/EPIR-258?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .venv/Scripts/Tramstops_API/__pycache__/buffer_exchange.cpython-312.pyc
Comment (gh-aabukhouski) [inline .venv/Scripts/Tramstops_API/__pycache__/buffer_exchange.cpython-312.pyc:1]: I recommend don't check in binary files into github.

File: .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange.log
Comment (gh-aabukhouski) [inline .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange.log:1]: the same comment for log files as for binary pyc - I wouldn't push it to github.

File: .venv/Scripts/__pycache__/hamilton.cpython-312.pyc
Comment (gh-aabukhouski) [inline .venv/Scripts/__pycache__/hamilton.cpython-312.pyc:1]: I recommend don't check in binary files into github.

File: .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange_for_one_batch.log
Comment (gh-aabukhouski) [inline .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange_for_one_batch.log:1]: Same as above

File: .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange_for_two_batches.log
Comment (gh-aabukhouski) [inline .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange_for_two_batches.log:1]: Same comment as above for log files

**Review by gh-aabukhouski — COMMENTED** [review]
Left couple comments. Why did you put Scripts into ".venv" folder?  :)

File: .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange_for_one_batch.log
Comment (gh-dkale) [reply .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange_for_one_batch.log:1]: Looks like the gitignore file wasn't pushed. When I did the initial commit, it added it but later I excluded the logs. Will push the gitignore file and delete the log files from here.

File: .venv/Scripts/__pycache__/hamilton.cpython-312.pyc
Comment (gh-dkale) [reply .venv/Scripts/__pycache__/hamilton.cpython-312.pyc:1]: Looks like the gitignore file wasn't pushed. When I did the initial commit, it added it but will push the gitignore file and delete the files from here.

File: .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange.log
Comment (gh-dkale) [reply .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange.log:1]: Looks like the gitignore file wasn't pushed. When I did the initial commit, it added it but will push the gitignore file and delete the files from here.

File: .venv/Scripts/Tramstops_API/__pycache__/buffer_exchange.cpython-312.pyc
Comment (gh-dkale) [reply .venv/Scripts/Tramstops_API/__pycache__/buffer_exchange.cpython-312.pyc:1]: Looks like the gitignore file wasn't pushed. When I did the initial commit, it added it but will push the gitignore file and delete the files from here.

File: .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange_for_two_batches.log
Comment (gh-dkale) [reply .venv/Scripts/logs/TestCaseLogs/automation_test_buffer_exchange_for_two_batches.log:1]: Will push the gitignore file which will take care of it

**Pull request closed without merging**